### ISSUE TITLE
Accessibility for Card: Ensure keyboard focus is indicated visually

### ISSUE THREAD

**Issue by cjwest** [SU-SWS/decanter#406]
**Accessibility for Card**
***Ensure keyboard focus is indicated visually (Severity: 7)***
The link "Button label", with style .su-button does not have a visible indication of focus in high contrast mode.

[User Impact]
Users with low vision who enable high contrast mode will not be able to track their location on the page and tell when this link has focus.

[Code Reference]
`<a href="" class="su-link su-button">Button Label</a>`

**To Reproduce**
1. Go to http://decanter.stanford.edu/section-components.html#kssref-components-card

**Expected behavior**

Provide a highly visible indication of focus that persists in high contrast mode. This can be done with CSS outline styles.

**Screenshots**
![image](https://user-images.githubusercontent.com/284440/56448294-c3394a80-62c2-11e9-8e08-6eb9afb9e199.png)

**Additional context**
https://stanford.levelaccess.net/public/reporting/view_module.php?module_id=9631562

**Comment by buttonwillowsix** [comment]
@kerri-augenstein when you get back, this needs your eyes.

**Comment by kerri-augenstein** [comment]
Hey @cjwest and @buttonwillowsix, 

I think the outline is great, and totally understand we need something that will be high contrast, so our typical light blue doesn't quite work. 

What do you think of using a traditional dark blue, instead of the red, as often red is an indicator of "incorrect" when filling out things like forms. 

Does #0000FF work? Could we add a slight blur around that line as well? Is something like that done with a drop shadow maybe?

**Comment by kgcreative** [comment]
See this post for consistent native browser styles. I think it might be a bit out of date, but windows high contrast mode should pick up on the styling anyway. It’s worth testing.

As much as possible, I would prefer to not change the native browser’s outline colors, since that messes with the mental models of keyboard and accessibility-needs users

https://ghinda.net/article/mimic-native-focus-css/

**Comment by yvonnetangsu** [comment]
Hi @kerri-augenstein , thanks for bringing this up! And hi, @kgcreative *waves* 😄 
I'll be playing John Holleman's role for now and for the design part of the next Decanterpalooza. I'd be happy to join you guys for any design related discussion if you need me 😃 Look forward to working together more!

**Comment by kerri-augenstein** [comment]
@kgcreative that sounds good to me, makes sense.

**Comment by yvonnetangsu** [comment]
Outlines have been added back to buttons
